app/v2/inbound_sms/get_inbound_sms.py

```diff
@@ -0,0 +1,83 @@
+import uuid
+
+from flask import jsonify, request, url_for, current_app
+from sqlalchemy.orm.exc import NoResultFound
+from werkzeug.exceptions import abort
+
+from notifications_utils.recipients import validate_and_format_phone_number
+from notifications_utils.recipients import InvalidPhoneError
+
+from app import authenticated_service
+from app.dao import inbound_sms_dao
+from app.v2.errors import BadRequestError
+from app.v2.inbound_sms import v2_inbound_sms_blueprint
+
+
+@v2_inbound_sms_blueprint.route("/<user_number>", methods=['GET'])
+def get_inbound_sms_by_number(user_number):
```

Comment:
Why is user_number a path param and not a query param?

Reply:
it's following a similar pattern to getting notification by id. Could convert it to a query param if that is preferable as I guess it is optional and generally path params are used for required params.

Reply:
It is preferable, plus we don't want the user_number showing up in our logs.